src/nidigital/system_tests/test_system_nidigital.py

```diff
@@ -558,6 +558,175 @@ def test_ppmu_source(multi_instrument_session):
+    # Tests configure_time_set_period()
+    # Tests get_time_set_period()
+    time_set_name = 'time_set_abc'
+    time_set_period = datetime.timedelta(microseconds=10)
+    multi_instrument_session.load_pin_map(os.path.join(test_files_base_dir, "pin_map.pinmap"))
+
+    multi_instrument_session.create_time_set(time_set_name)
+    assert multi_instrument_session.get_time_set_period(time_set_name) == 1e-6
+    multi_instrument_session.configure_time_set_period(time_set_name, time_set_period)
+    assert multi_instrument_session.get_time_set_period(time_set_name) == time_set_period.total_seconds()
+
+
+def test_configure_get_time_set_drive_format(multi_instrument_session):
+    # Tests configure_time_set_drive_format()
+    # Tests get_time_set_drive_format()
+    time_set_name = 'time_set_abc'
+    time_set_drive_format = nidigital.DriveFormat.SBC
+    multi_instrument_session.load_pin_map(os.path.join(test_files_base_dir, "pin_map.pinmap"))
+
+    multi_instrument_session.create_time_set(time_set_name)
+    assert multi_instrument_session.pins['site0/PinA', 'site1/PinC'].get_time_set_drive_format(time_set_name) == nidigital.DriveFormat.NR
+    multi_instrument_session.pins['site0/PinA', 'site1/PinC'].configure_time_set_drive_format(time_set_name, time_set_drive_format)
+    assert multi_instrument_session.pins['site0/PinA', 'site1/PinC'].get_time_set_drive_format(time_set_name) == time_set_drive_format
+
+
+def test_configure_get_time_set_edge(multi_instrument_session):
+    # Tests configure_time_set_edge()
+    # Tests get_time_set_edge()
+    time_set_name = 'time_set_abc'
+    time_set_period = datetime.timedelta(microseconds=10)
+    time_set_drive_on = time_set_period * 0.5
+    multi_instrument_session.load_pin_map(os.path.join(test_files_base_dir, "pin_map.pinmap"))
+
+    multi_instrument_session.create_time_set(time_set_name)
+    multi_instrument_session.configure_time_set_period(time_set_name, time_set_period)
+    assert multi_instrument_session.pins['site0/PinA', 'site1/PinC'].get_time_set_edge(time_set_name, nidigital.TimeSetEdgeType.DRIVE_ON) == 0
+    multi_instrument_session.pins['site0/PinA', 'site1/PinC'].configure_time_set_edge(time_set_name, nidigital.TimeSetEdgeType.DRIVE_ON, time_set_drive_on)
+    assert multi_instrument_session.pins['site0/PinA', 'site1/PinC'].get_time_set_edge(time_set_name, nidigital.TimeSetEdgeType.DRIVE_ON) == time_set_drive_on.total_seconds()
+
+
+def test_time_set_edges(multi_instrument_session):
+    # Tests configure_time_set_drive_edges()
+    # Tests configure_time_set_compare_edges_strobe()
+    time_set_name = 'time_set_abc'
+    time_set_period = datetime.timedelta(microseconds=10)
+    time_set_drive_format = nidigital.DriveFormat.RL
+    time_set_drive_on = time_set_period * 0.1
+    time_set_drive_data = time_set_period * 0.2
+    time_set_drive_return = time_set_period * 0.8
+    time_set_drive_off = time_set_period * 0.9
+    time_set_strobe_edge = time_set_period * 0.5
```

Comment:
`time_set_strobe` to be consistent with other names

Reply:
Done.